frontend/src/libs/components/modal-window/modal-window.tsx

```diff
@@ -0,0 +1,24 @@
+import ReactDOM from 'react-dom';
+
+import { IconButton } from '~/libs/components/components.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+
+import styles from './styles.module.scss';
+
+const ModalWindow: React.FC = () => {
+  return ReactDOM.createPortal(
+    <>
+      <div className={styles.backdrop} />
+      <div className={styles.modal}>
+        <IconButton
+          icon="closePopUp"
+          label="Close modal"
+          className={getValidClassNames('', styles.modalPopUpClose)}
```

Comment:
```suggestion
          className={styles.modalPopUpClose}
```

Reply:
In this case I get lint:type error 
![image](https://user-images.githubusercontent.com/87016333/226578515-0c030999-6609-4f81-bb87-0e2159eac540.png)

Reply:
Should I add like "as string"?

Reply:
`styles.modalPopUpClose as string`